Framework/Mobile/LibTest/AppDelegate.m

```diff
@@ -1,41 +0,0 @@
-#import "AppDelegate.h"
-#import "ViewController.h"
-
-#import <CocoaLumberjack/CocoaLumberjack.h>
-#import "Formatter.h"
```

Comment:
It's a good idea to remove this from the framework, but we should move it to Demos.